### ISSUE TITLE
Components invoked with {{ name (notice the space) aren't found

### ISSUE THREAD

**Issue by gossi** [rajasegar/eax#8]
I did run eax for our app. Some components showed up as not being used. When looking into, that is because they contain a space when being invoked, like this:

```hbs
{{ component-name }}
```

I guess that might be it?

PS. This is very old code - yes ;)

**Comment by rajasegar** [comment]
Yes @gossi It was due to the regex pattern matching here. Ideally we should be parsing the template files and find out the components, but this will take a lot of time to process. That's why I went with the regex approach, looks like need to tweak the  regex pattern here.
https://github.com/rajasegar/eax/blob/a5985fd7462b4295819ee0fee69af5d1e105a396/src/pages/components.js#L154
